boards/arc/hsdk/doc/index.rst

```diff
@@ -0,0 +1,138 @@
+.. _iotdk:
```

Comment:
label names must be unique across all our published docs.  How about instead:

```
.. _hsdk:
```

Reply:
fixed